lib/codegen/recorder.dart

```diff
@@ -257,7 +257,8 @@ class Recorder {
     // [node] is the initialization expression, we walk up to get to the actual
     // member declaration where the metadata is attached to.
     while (node is! ClassMember) node = node.parent;
-    return node.metadata.map(_convertAnnotation).toList();
+    return (node as ClassMember).metadata.map(_convertAnnotation).toList() as
```

Comment:
I can add the generic, but the analyzer complains unless I also include the cast.